src/main.rs

```diff
@@ -241,6 +242,77 @@ fn main() -> Result<()> {
 
             Ok(())
         }
+        Some(("qubot", args)) => {
```

Comment:
This could be merged with the `model` command above and change it to a `--qubot` or `-q` flag instead. We can of course do this later as a follow-up.

Reply:
yes I can add this :)